XML/TestCases/PerformanceTests.xml

```diff
@@ -770,4 +770,42 @@
       <OverrideVMSize>Standard_D3_v2</OverrideVMSize>
     </SetupConfig>
   </test>
+  <test>
+    <testName>PERF-APACHE-BENCHMARK</testName>
+    <testScript>PERF-APACHE-BENCHMARK.ps1</testScript>
+    <runSetupScriptOnlyOnce>yes</runSetupScriptOnlyOnce>
+    <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\perf_apache.sh</files>
+    <TestParameters>
+      <param>PERF_NIC=HYPERV_PERF_NIC_NAME</param>
+    </TestParameters>
+    <Platform>Azure,HyperV</Platform>
+    <Category>Performance</Category>
+    <Area>MIDDLEWARE</Area>
+    <Tags>middleware,apache,sriov</Tags>
+    <Priority>1</Priority>
```

Comment:
why is it '1'? Please share your justification here.

Reply:
Thank you for your comments. I think "2" is appropriate.

Reply:
Fixed